packages/web/src/algo-signer-types.ts

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Can you please add link(s) to top of file, from where we have taken these typings?

Reply:
Yes